packages/beacon-node/src/chain/validation/blobSidecar.ts

```diff
@@ -14,21 +19,25 @@ import {RegenCaller} from "../regen/index.js";
 export async function validateGossipBlobSidecar(
   chain: IBeaconChain,
   blobSidecar: deneb.BlobSidecar,
-  subnet: number
+  subnet: number,
+  fork: ForkName
 ): Promise<void> {
   const blobSlot = blobSidecar.signedBlockHeader.message.slot;
 
   // [REJECT] The sidecar's index is consistent with `MAX_BLOBS_PER_BLOCK` -- i.e. `blob_sidecar.index < MAX_BLOBS_PER_BLOCK`.
-  if (blobSidecar.index >= chain.config.MAX_BLOBS_PER_BLOCK) {
+  const maxBlobsPerBlock = isForkPostElectra(fork)
```

Comment:
Assigning `maxBlobsPerBlock` is done in multiple places, we could think about adding a helper for that as there will be likely more values in the future since we intend to bump the blob count further (likely in the next hard fork)

Maybe something similar to what the CL spec uses in their tests [get_max_blob_count](https://github.com/ethereum/consensus-specs/blob/e6bddd966214a19d2b97199bbe3c02577a22a8b4/tests/core/pyspec/eth2spec/test/helpers/blob.py#L109)

Reply:
Added `constantsHelper.ts` for the purpose of reducing number of conditional assignments in the codebase.

Can expand the scope of `constantsHelper` to outside of blobs in another PR